frontend/src/hooks/useWidgetSorting.js

```diff
@@ -22,10 +22,13 @@ export default function useWidgetSorting(
 ) {
   const [sortConfig, setSortConfig] = useSessionSort(defaultSortConfig, localStorageKey);
 
-  const requestSort = useCallback((sortBy) => {
+  const requestSort = useCallback((sortBy, passedDirection = null) => {
     // Get sort direction.
     let direction = 'asc';
-    if (
+    if (passedDirection) {
+      // If the direction is passed, use it.
+      direction = passedDirection;
+    } else if (
```

Comment:
Can you leave a comment explaining why we need to optionally override the sortConfig direction?

Reply:
Just seems like you could mutate the object that already has the direction in it instead of adding another parameter and all this additional logic

Reply:
I added a comment.

This seems to just take the opposite of whats currently set in the sortConfig (arrow sort).